inst/components/scss/sidebar.scss

```
@@ -65,15 +67,13 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
     padding: var(--_padding);
-    transition: padding var(--_transition-easing-x) var(--_transition-duration);
+    transition: padding var(-- _transition-easing-x) var(--_transition-duration);
```

Was this change intentional?

---

Weird! No, not sure how that snuck in there.

```suggestion
    transition: padding var(--_transition-easing-x) var(--_transition-duration);
```